test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -107,18 +106,18 @@ type testConfig struct {
 	numNodes            int
 	mutatedNodeTemplate *v1.Node
 	mutatedPodTemplate  *v1.Pod
-	schedulerSupport    *factory.ConfigFactoryArgs
+	clientset           clientset.Interface
```

Comment:
This should be replaced with `config.PodInformer` to fetch pods.

Reply:
done